src/main/java/org/folio/service/pieces/PieceStorageService.java

```diff
@@ -185,4 +187,9 @@ private Future<List<Piece>> getPieceChunkByLineIds(Collection<String> poLineIds,
       .map(PieceCollection::getPieces);
   }
 
+  private static boolean shouldFilterPiecesForTenant(String centralTenantId, RequestContext requestContext) {
+    var requestTenantId = TenantTool.tenantId(requestContext.getHeaders());
+    return requestTenantId.equals(centralTenantId);
```

Comment:
Is this to make pieces viewable only by the central tenant?

Reply:
To match condition when to filter fetched pieces while calling GET all pieces endpoint